playout/libretime_playout/player/fetch.py

```diff
@@ -267,66 +199,15 @@ def update_liquidsoap_connection_status(self):
 
     @ls_timeout
     def update_liquidsoap_stream_format(self, stream_format):
-        # Push stream metadata to liquidsoap
-        # TODO: THIS LIQUIDSOAP STUFF NEEDS TO BE MOVED TO PYPO-PUSH!!!
-        try:
-            self.telnet_lock.acquire()
-            tn = telnetlib.Telnet(
-                self.config.playout.liquidsoap_host,
-                self.config.playout.liquidsoap_port,
-            )
-            command = ("vars.stream_metadata_type %s\n" % stream_format).encode("utf-8")
-            logger.info(command)
-            tn.write(command)
-            tn.write(b"exit\n")
-            tn.read_all()
-        except Exception as exception:
-            logger.exception(exception)
-        finally:
-            self.telnet_lock.release()
+        self.liq_client.settings_update(message_format=stream_format)
```

Comment:
All those changes unwrap liquidsoap calls from the broad exception handling, I wonder how and where this should be handled.

We should probably catch any ConnectionError and TimeoutError.

I don't know if we should log and ignore or fail hard ? Current behavior is log and ignore. 

Where this should be handled ? Maybe in the client context __exit__ exception handling.